src/java/picard/analysis/GcBiasMetricsCollector.java

```diff
@@ -246,6 +252,26 @@ public void addMetricsToFile(final MetricsFile<GcBiasMetrics, Integer> file) {
     }
 
     /////////////////////////////////////////////////////////////////////////////
+    // Calculates the normalized coverage over each gc content quintile
+    /////////////////////////////////////////////////////////////////////////////
+    private double calculateGcNormCoverage(final double meanReadsPerWindow, final int[] readsByGc, final int start, final int end) {
+        int windowsTotal = 0;
+        double sum = 0.0;
+        for (int i = start; i <= end; i++) {
+            if (windowsByGc[i] != 0) {
+                final double norm = ((double) readsByGc[i] / windowsByGc[i]) / meanReadsPerWindow;
```

Comment:
you seem to be dividing by windowsByGc and then multiplying by it again in the next line...can this be avoided?

Also meanReadsPerWindow is a constant, can you divide by it in the end of the loop?